rubicon_ml/client/project.py

```diff
@@ -347,6 +352,84 @@ def dataframes(self, tags=[], qtype="or", recursive=False, name=None):
 
         return self._dataframes
 
+    @failsafe
+    def archive(self, experiment_names: Optional[list[str]] = None):
```

Comment:
I think we should ultimately just take in experiment objects here instead of names or IDs. then users can just get them by name or ID or tags or whatever they want before the send them in

Reply:
we may also want to consider a `remote_root` option similar to `experiments_from_archive`'s for this function as well. that way we could facilitate a workflow thats something more like 

```
write individual experiments locally -> 
archive to S3 -> 
later read archives from S3 back to local and unzip
``` 

instead of 

```
write individual experiments locally -> 
archive locally -> 
sync everything to S3 -> 
later read archives from S3 back to local and unzip
```